src/Bridge/Symfony/Routing/Router.php

```diff
@@ -30,11 +30,20 @@
         UrlGeneratorInterface::NET_PATH => RouterInterface::NETWORK_PATH,
     ];
 
+    const CONFIG_MAP = [
+        'ABS_URL' => UrlGeneratorInterface::ABS_URL,
+        'ABS_PATH' => UrlGeneratorInterface::ABS_PATH,
+        'REL_PATH' => UrlGeneratorInterface::REL_PATH,
+        'NET_PATH' => UrlGeneratorInterface::NET_PATH,
+    ];
+
     private $router;
+    private $defaultReferenceType;
 
-    public function __construct(RouterInterface $router)
+    public function __construct(RouterInterface $router, string $defaultReferenceType)
```

Comment:
Well it does not have to be a constant CONFIG, but in my opinion we need a way to let the caller of the function overwrite the default settings.
I could also use $referenceType = null in the method signature.
How else would you solve this?